src/Cache/Engine/RedisEngine.php

```diff
@@ -53,10 +56,19 @@ class RedisEngine extends CacheEngine
      * - `server` URL or IP to the Redis server host.
      * - `timeout` timeout in seconds (float).
      * - `unix_socket` Path to the unix socket file (default: false)
+     * - `readTimeout` Read timeout in seconds (float).
+     * - `nodes` URL or IP addresses of the Redis cluster nodes.
+     *   Format: an array of strings in the form `<ip>:<port>`, like:
+     *   [
+     *       '<ip>:<port>',
+     *       '<ip>:<port>',
+     *       '<ip>:<port>',
+     *   ]
      *
      * @var array<string, mixed>
      */
     protected array $_defaultConfig = [
+        'name' => null,
```

Comment:
Should we name this key as `clusterName` to be more obvious about it's purpose?

Reply:
Yes, in the RedisCluster file the context was clear but now it isn't so I will update it